introduction.md

```diff
@@ -0,0 +1 @@
+This is RajyaLakshmi02. This task is done to complete the tutorial episode2. This is the first course in code school it is the great session I am going through. If i see for Cucumber, Jenkins it will be good to me. This is for earning incomplete Badge. My favouriate code school path is HTML, Java.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.